kitchen-tests/cookbooks/end_to_end/recipes/_cron.rb

```diff
@@ -56,12 +56,15 @@
 # cron resource
 #
 
-cron "some random cron job" do
-  minute  0
-  hour    23
-  command "/usr/bin/true"
-end
+# Opensuse leap doesn't have a crontab till a user creates one causing these resources to fail
+unless platform_family?("suse")
```

Comment:
Instead of nesting, this instead?
```suggestion
return unless platform_family?("suse")
```

Reply:
done thanks